pkg/build/apis/build/types.go

```diff
@@ -304,6 +304,9 @@ type BuildStatus struct {
 	// including start time, duration (in milliseconds), and the steps that
 	// occured within each stage.
 	Stages []StageInfo
+
+	// LogTail is the last few lines of the build log.  This value is only set for builds that failed.
+	LogTail *string
```

Comment:
discussed with jordan, let's call this `excerpt` or `outputExcerpt`.  No guarantee there is a log.  Also, not a pointer to string, just do omit empty and a normal string.  Pointer to string for this sort of field is not necessary.